src/patternfly/components/Chips/chip-group.scss

```diff
@@ -0,0 +1,44 @@
+@import "../../patternfly-utilities";
+
+.pf-c-chip-group {
+  --pf-c-chip-group--PaddingTop: calc(#{var(--pf-global--spacer--xs)} / 2);
+  --pf-c-chip-group--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-chip-group--PaddingBottom: calc(#{var(--pf-global--spacer--xs)} / 2);
+  --pf-c-chip-group--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-chip-group--BorderRadius: var(--pf-global--BorderRadius--sm);
+  --pf-c-chip-group--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-chip-group--MarginBottom: var(--pf-global--spacer--sm);
+  --pf-c-chip-group--m-toolbar--BackgroundColor: var(--pf-global--BackgroundColor--300);
+
+  // Label in group
+  --pf-c-chip-group__label--PaddingLeft: var(--pf-global--spacer--xs);
+  --pf-c-chip-group__label--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-chip-group__label--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-chip-group__label--Maxwidth: #{pf-size-prem(120px)}; // Hard coded
+
+  display: inline-flex;
+  flex-wrap: wrap;
+  align-items: center;
+  padding-top: var(--pf-c-chip-group--PaddingTop);
+  padding-right: var(--pf-c-chip-group--PaddingRight);
+  padding-bottom: var(--pf-c-chip-group--PaddingBottom);
+  padding-left: var(--pf-c-chip-group--PaddingLeft);
+  margin-right: var(--pf-c-chip-group--MarginRight);
```

Comment:
Components should not margin themselves because by definition, they are supposed to take all available space.